lib/Target/RISCV/RISCVRelocationInternal.h

```diff
@@ -67,10 +67,9 @@ enum : uint32_t {
   QUALCOMMVendorRelocationOffset =
       FirstQUALCOMMVendorRelocation - FirstNonstandardRelocation,
 
-#define ELF_RISCV_NONSTANDARD_RELOC(vendor_symbol, name, value)                \
-  name = value + vendor_symbol##VendorRelocationOffset,
+#define ELF_RISCV_NONSTANDARD_RELOC_QUALCOMM(name, value)                \
+  name = value + QUALCOMMVendorRelocationOffset,
```

Comment:
It seems that `RISCV_nonstandard.def` still expects `ELF_RISCV_NONSTANDARD_RELOC` macro to be defined. 

If we need both `ELF_RISCV_NONSTANDARD_RELOC` and `ELF_RISCV_NONSTANDARD_RELOC_QUALCOMM`, then
does the suggested change seems right?

```suggestion
#define ELF_RISCV_NONSTANDARD_RELOC(vendor_symbol, name, value)                \
  name = value + vendor_symbol##VendorRelocationOffset
  
#define ELF_RISCV_NONSTANDARD_RELOC_QUALCOMM(name, value)                \
  ELF_RISCV_NONSTANDARD_RELOC(QUALCOMM, name, value)
```

Reply:
Please ignore my above comment, the linked llvm-project PR removes all usage of `ELF_RISCV_NONSTANDARD_RELOC` macros.